Objects/genericaliasobject.c

```diff
@@ -121,6 +121,39 @@ ga_repr_item(_PyUnicodeWriter *writer, PyObject *p)
     return err;
 }
 
+static int
+ga_repr_items_list(_PyUnicodeWriter *writer, PyObject *p)
+{
+    assert(PyList_CheckExact(p));
+
+    Py_ssize_t len = PyList_GET_SIZE(p);
+
+    if (_PyUnicodeWriter_WriteASCIIString(writer, "[", 1) < 0) {
+        goto error;
```

Comment:
As the error label is just `return -1`, I don't think we need the goto. The `goto error` pattern is useful if there is a lot of items to DECREF in the error path.

Reply:
(please, don't share my secret with anybody, but: I love `goto` statements).

Refactored! Thanks for the review :)

Reply:
https://xkcd.com/292/